Caps the number of active sessions (#8540)
MicrosoftDocs/sql-docs: azure-sql/database/xevent-db-diff-from-svr.md

- Run fewer concurrent event sessions.
- Through your **CREATE** and **ALTER** statements for event sessions, reduce the amount of memory you specify on the **MAX\_MEMORY** clause.

> There is a cap on number of active XEvent sessions in Azure SQL Database:

Reviewer: ```suggestion
There is a cap on number of started XEvent sessions in Azure SQL Database:
```

  - In single Azure SQL Database, the limit is 100.

>   - In an elastic pool, the number of active Xevent sessions of all databases is 100.

Reviewer: ```suggestion
  - In an elastic pool, the limit is 100 database-scoped sessions per pool.
 
In [dense elastic pools](elastic-pool-resource-management.md), starting a new extended event session may fail due to memory constraints even when the total number of started sessions is below 100.
```

- Run fewer concurrent event sessions.
- Through your **CREATE** and **ALTER** statements for event sessions, reduce the amount of memory you specify on the **MAX\_MEMORY** clause.

There is a cap on number of started XEvent sessions in Azure SQL Database:

>   - In single Azure SQL Database, the limit is 100.

Reviewer 2: ```suggestion
  - In a single Azure SQL Database, the limit is 100.
```
